core/templates/dev/head/pages/admin/admin_navbar_directive.html

```diff
@@ -1,14 +1,15 @@
 <script type="text/ng-template" id="admin/navbar">
   <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
-    <div class="navbar-container" style="background-color: #00376d">
+    <div class="navbar-container">
       <div class="navbar-header protractor-test-navbar-header pull-left">
         <a class="oppia-navbar-brand-name oppia-transition-200" href="/library">
           <img ng-src="<[logoWhiteImgUrl]>" class="oppia-logo" ng-class="'oppia-logo-wide'">
         </a>
         <ul class="nav navbar-nav oppia-navbar-breadcrumb">
           <li>
             <span class="oppia-navbar-breadcrumb-separator"></span>
-            Admin
+            <!-- IMPORTANT: The change to this text should be reverted prior to merge into develop. -->
```

Comment:
Tried, too many things are IMPORTANT apparently (40 other matches). I addressed this one, though, and did a brief search for similar HTML comments (none found).

Removed this one, though.